components/stripe/actions/cancel-or-reverse-payout/cancel-or-reverse-payout.js

```diff
@@ -0,0 +1,35 @@
+const stripe = require("../../stripe.app.js");
+
+module.exports = {
+  key: "stripe-cancel-or-reverse-payout",
+  name: "Cancel Or Reverse a Payout",
+  version: "0.0.1",
+  description: "Cancel or reverse a [payout](https://stripe.com/docs/payouts). " +
+    "A payout can be canceled only if it has not yet been paid out. A payout can be reversed " +
+    "only if it has already been paid out. Funds will be refunded to your available balance.",
+  props: {
+    stripe,
+    id: {
+      propDefinition: [
+        stripe,
+        "payout",
+      ],
+      required: true,
```

Comment:
This is the default, so you can remove and the property is technically `optional` (`false` by default): https://pipedream.com/docs/components/api/#user-input-props